src/components/Project/Project.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+
+import { Box, Card, CardContent, Link, Typography } from '@mui/material'
+import {} from 'react'
+import { Link as LinkInterface } from '../../app/dataModels/Link'
+import { Project } from '../../app/dataModels/Project'
```

Comment:
Hello @CorreyL , sorry I know this was a long time ago, but right now I am having this error on my VSCode:
![image](https://github.com/OpenBCca/openbc-web/assets/72838071/a720ddad-511e-4b9a-b422-59bc0690f7ad)

After debugging, it turns out that **Project** is a common name, so after I change it to ProjectInterface, the error is gone.